src/docs/product/dashboards/widget-library/index.mdx

```diff
@@ -19,7 +19,7 @@ The library includes the following widgets:
 - **LCP by Country**: World map showing the p75 of page load times for each country
 - **Miserable Users**: The total number of unique users who have experienced slow transactions (transaction duration greater than 1200ms)
 - **Slow vs Fast Transactions**: Bar chart comparing the percentage of transactions that are over 300ms (slow) and under 300ms (fast)
-- **Latest Unresolved Issues**: A table of unresolved issues ordered by the most recently seen issues
+- **Issues For Review**: A table of unresolved issues for you or your team's review ordered by the most recently seen issues
```

Comment:
might have to be either `for your or your team's review` or `for you or your team to review`?

Reply:
english hard 😃 

updated to `for you or your team to review`!